iceoryx_posh/test/moduletests/test_mq_message.cpp

```diff
@@ -430,4 +430,13 @@ TEST_F(MqMessage_test, setMessage)
     message1.setMessage("1,2,3,4");
     EXPECT_THAT(message1.isValid(), Eq(false));
 }
+
+TEST_F(MqMessage_test, CheckEqualMessageOperatorReturnsTrue)
+{
+    MqMessage message1;
+    MqMessage message2;
+    message1.setMessage("asd1,asd2,asd3,asd4,");
+    message2.setMessage("asd1,asd2,asd3,asd4,");
+    EXPECT_TRUE(message1 == message2);
```

Comment:
We should always test the negative case as well, what if the operator always returns true? (The test would pass as well...)

Reply:
Yeah , the negative case will be added